src/Generator/Agoda.IoC.Generator/Agoda.IoC.Generator.csproj

```diff
@@ -17,26 +17,23 @@
 		<RepositoryUrl>https://github.com/agoda-com/Agoda.IoC</RepositoryUrl>
 		<PackageId>Agoda.Ioc.Generator</PackageId>
 		<Description>Share dotnet C# IoC source generator, used at Agoda.</Description>
-
 	</PropertyGroup>
-
 	<ItemGroup>
 		<PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
 		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.5.0" PrivateAssets="all" />
-	</ItemGroup>
-
+		<PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.1.0" />
```

Comment:
Why are these needed for a documentation change?

Reply:
removed